client/components/productDetail.js

```diff
@@ -38,6 +38,7 @@ class ProductDetail extends Component {
       return <h1>loading...</h1>
     } else {
       let product = this.props.product
+      console.log(product.reviews)
```

Comment:
let's remove any console logs before merging to master

Reply:
removed